### ISSUE TITLE
Recording output from sound card

### ISSUE THREAD

**Issue by cauli** [gillesdemey/node-record-lpcm16#2]
Hey, thanks for this module, it is really useful!

This is a feature request, I am making a sound player directly on node's CLI using play.js (mpg123) and I needed to record the output of the sound card to a sound file of any type.

One way to do that would be a hardware hack of placing a P2-P2 cable on the output plug (headphones) to the input plug (Mic), but that would be kind of too hacky and I would really like to play the sound through the speakers while recording it. 

Do you have any ideas?

**Comment by Glavin001** [comment]
Maybe use something like [Soundflower](https://rogueamoeba.com/freebies/soundflower/) and change your sound output device to `Soundflower` and then your input device to `Soundflower` so your output actually gets routed back as microphone input, which you could then record.

**Comment by cauli** [comment]
Thanks for the help @Glavin001.

I had thought of using Soundflower at the time I had encountered this problem, it is a nice solution in **most** cases.

I ended up using a cheap USB adapter like this with the hardware trick and all went well ![Usb audio adapter](http://www.geekalerts.com/u/usb-21-audio.jpg).

The problem with Soundflower is that I needed the program to save long audios for an entire day in realtime and Soundflower is known to introduce delays because of the audio buffer or something. 
I also needed it to run on Linux.

The hardware trick wasn't sooo hacky after all (:
